styles/widgets/generic/color-schemes/contrast/generic.contrast.less

```diff
@@ -48,14 +48,20 @@
 @typography-link-color: @base-link-color;
 @base-disabled-opacity: 0.5;
 
-// Validation
```

Comment:
This change is not clear to me

Reply:
@base-invalid-faded-border-color was moved to the base section (above Validation). Git diff here is not clear.